### PR TITLE
Added testing to Affinity page unauth testing

### PR DESCRIPTION
**Pull request by alvaradom0321**
## Describe context / purpose for this PR
Add test testing Request an Affinity Group link takes you to login and then a webform to request a new Affinity Group | affinity-group-unauth

## Issue link
https://cyberteamportal.atlassian.net/browse/D8-1598

## Any other related PRs?
N/A

## Link to MultiDev instance

http://md-{{d8-1598 }}-accessmatch.pantheonsite.io

## Checklist for PR author
- [ ] I have checked that the PR is ready to be merged
- [x] I have reviewed the DIFF and checked that the changes are as expected
- [x] I have assigned myself or someone else to review the PR

### CODE REVIEW

File: tests/behat/features/asp/affinity-groups-unauth.feature [hunk 1]
```diff
@@ -30,3 +30,11 @@ Feature: test ACCESS Support Affinty Groups Page
     Then I should be on "/user/login"
 
 #TODO: test Request an Affinity Group link takes you to login and then a webform to request a new Affinity Group
+    When I am on "/affinity_groups"
+    Then I should see "submit a request form"
+    When I click "submit a request form"
+    Then I should be on "user/login?destination=/form/affinity-group-request"
+    #TODO: test webform to request after you login
```
[the remark this answers is not in the document]
Comment (laheara143) [reply tests/behat/features/asp/affinity-groups-unauth.feature:37]: @alvaradom0321

[the remark this answers is not in the document]
Comment (alvaradom0321) [reply tests/behat/features/asp/affinity-groups-unauth.feature:37]: @laheara143 
#TODO: test webform to request after you login
this means that once you click “submit a request “ it takes you to log in but the to do is we need to test that when we log in it takes us to the webform to submit a request

[the remark this answers is not in the document]
Comment (laheara143) [reply tests/behat/features/asp/affinity-groups-unauth.feature:37]: Okay, I think [affinity-groups-auth.feature] has that part so no worries. I will double check that file to make sure

**Review by laheara143 — APPROVED** [review]

**Comment by jasperlieber** [comment]
redone on https://github.com/necyberteam/cyberteam_drupal/pull/755

**Pull request closed without merging**